service/src/test/resources/data/incoming_requests/mocked_request2c_shared_patient.json

```diff
@@ -0,0 +1,82 @@
+{
+  "requestId": "22022_CD",
+  "recipe": "GENESET101",
+  "projectManagerName": "",
+  "piEmail": "",
+  "labHeadName": "James Holt",
+  "labHeadEmail": "holtj@mskcc.org",
+  "investigatorName": "Jessie Hall",
+  "investigatorEmail": "hallj@mskcc.org",
+  "dataAnalystName": "",
+  "dataAnalystEmail": "",
+  "otherContactEmails": "preclinical_group_shared@mskcc.org",
+  "dataAccessEmails": "",
+  "isCmoRequest":true,
+  "bicAnalysis": false,
+  "qcAccessEmails": "",
+  "pooledNormals": [
+    "/FASTQ/Project_POOLEDNORMALS/Sample_FFPEPOOLEDNORMAL_IGO_GENESET101_TGCTAGCT/FFPEPOOLEDNORMAL_IGO_GENESET101_TGCTAGCT_S118_R1_001.fastq.gz",
+    "/FASTQ/Project_POOLEDNORMALS/Sample_FFPEPOOLEDNORMAL_IGO_GENESET101_TGCTAGCT/FFPEPOOLEDNORMAL_IGO_GENESET101_TGCTAGCT_S118_R2_001.fastq.gz",
+    "/FASTQ/Project_POOLEDNORMALS/Sample_FROZENPOOLEDNORMAL_IGO_GENESET101_GCGGCTAG/FROZENPOOLEDNORMAL_IGO_GENESET101_GCGGCTAG_S22_R1_001.fastq.gz",
+    "/FASTQ/Project_POOLEDNORMALS/Sample_FROZENPOOLEDNORMAL_IGO_GENESET101_GCGGCTAG/FROZENPOOLEDNORMAL_IGO_GENESET101_GCGGCTAG_S22_R2_001.fastq.gz"
+  ],
+  "projectId": "22022",
+  "samples": [
+    {
```

Comment:
there appears to only be one sample but the case that we want to test for is at least 2 samples with the same patient id and same new request

Reply:
changes:
- different request id and project id that we do not already have in our mock datasets (make sure they start w/same pattern)
- add 2 more samples to this `samples` list, total should be 3 sample manifest jsons in this list
- all 3 samples share the same cmo patient id
- all 3 samples have different igo ids and cmo sample names